src/racingCar/CarRacing.js

```diff
@@ -0,0 +1,59 @@
+import { Console } from "@woowacourse/mission-utils";
+import Car from "./Car.js";
+import HandleIo from "./HandleIo.js";
+import ErrorChecker from "../checker/ErrorChecker.js";
+
+class CarRacing {
+    async run() {
+
+        const handleinput = new HandleIo();
+        const { nameInput, tryCount } = await handleinput.getInput();
+
+        this.startRacing(nameInput, tryCount);
+    };
+
+    startRacing(nameInput, tryCount) {
+
+        const cars = this.createCar(nameInput);
+
+        if (!cars) return;
+
+        ErrorChecker.checkTryCount(tryCount);
+
+        for (let i = 0; i < tryCount; i++) {
+            cars.forEach(car => car.move());
+            Console.print("\n");
+        };
+
+        this.checkWinner(cars);
+    };
+
+    createCar(input) {
+        const cars = input.split(',').map((name) => {
+            name = name.trim();
+            ErrorChecker.checkName(name);
+            return new Car(name);
+        });
+        return cars;
+    };
+
+    checkWinner(cars) {
+        let bestScore = 0;
+        let winner = [];
+
+        for (let i = 0; i < cars.length; ++i) {
+            const car = cars[i];
+            if (car.position > bestScore) {
+                bestScore = car.position;
+                winner = [car.name];
+            } else if (car.position === bestScore) {
+                winner.push(car.name);
+            };
+        };
```

Comment:
이렇게 하면 if else 분기를 if만으로 간단하게 처리할 수 있지 않을까 싶어요!
```suggestion
        for (let i = 0; i < cars.length; ++i) {
            const car = cars[i];
            if (car.position > bestScore) {
                bestScore = car.position;
                winner = [];
            } 
            
            if (car.position === bestScore) {
                winner.push(car.name);
            };
        };
```

Reply:
현재 이터레이션에서 계산하고있는게,
1. 최고 스코어 구하기
2. 최고 스코어인 승자(들) 구하기
이렇게 두가지 인것 같아요. 이를 분리해냈으면 좋겠다는 생각이 들어요.

```suggestion
  const bestScore = Math.max(...car.map(car => car.position));
  const winner = car.filter(car => car.position === bestScore).map(car => car.name);
```